portfolio/src/main/webapp/index.html

```diff
@@ -103,10 +103,11 @@ <h6 class="title"> Research Assistant Intern </h6>
     </div>
     <div class="map" id="map"> </div>
     <div class="comments-title"> Check out people's comments! </div>
-    <div>
+    <div class="comments-container">
       <span> I'd like to see </span> 
-      <input type="text" id="number" oninput="getComments()">
-      <span for="number"> comments. </span>
+      <input type="text" id="number" >
```

Comment:
I think you mentioned this during our meeting but the `id` here should be more specific. Make sure to signify that it's the number of comments to be displayed.

Reply:
Forgot to change it, thanks for the catch!